pkg/install/v1alpha1/cstor_volume.go

```diff
@@ -528,6 +528,11 @@ spec:
             ports:
             - containerPort: 3260
               protocol: TCP
+            env:
+            - name: QueueDepth
+              value: {{ .Config.QueueDepth.value }}
+            - name: Luworkers
+              value: {{ .Config.Luworkers.value }}
```

Comment:
I feel this will error out since Luworkers is not defined as a property of Config

Reply:
How can one determine the best value of Luworkers in one's environment?
Is there any default value that can be recommended?

Reply:
Same as above comment !